core/src/main/scala/org/apache/spark/rapids/tool/benchmarks/Benchmark.scala

```diff
@@ -99,18 +99,31 @@ class Benchmark(
 
     val firstBest = results.head.bestMs
     // The results are going to be processor specific so it is useful to include that.
-    out.println(RuntimeUtil.getJVMOSInfo.mkString("\n"))
+    val jvmInfo = RuntimeUtil.getJVMOSInfo
+    out.printf(s"%-26s ->   %-30s \n","JVM Name", jvmInfo("jvm.name"))
+    out.printf(s"%-26s ->   %-30s \n","Java Version",jvmInfo("jvm.version"))
+    out.printf(s"%-26s ->   %-30s \n","OS Name",jvmInfo("os.name"))
+    out.printf(s"%-26s ->   %-30s \n","OS Version",jvmInfo("os.version"))
+    out.printf(s"%-26s ->   %-30s \n","MaxHeapMemory", ("%6d" format Runtime.getRuntime.maxMemory()/1024/1024)+"MB")
```

Comment:
`%6d` will cause the string not to be aligned.

Reply:
Corrected